app/services/support_interface/work_history_break_export.rb

```diff
@@ -0,0 +1,221 @@
+      convert_seconds_to_months(time_in_seconds.sum)
+    end
+
+    def total_time_of_explained_breaks(explained_breaks)
+      time_in_seconds = explained_breaks.map { |work_history_break| work_history_break.end_date - work_history_break.start_date }
+      convert_seconds_to_months(time_in_seconds.sum)
+    end
+
+    def total_time_of_unexplained_breaks(unexplained_breaks)
+      unexplained_breaks.sum(&:length)
+    end
+
+    def total_time_volunteering_during_breaks(breaks, volunteering_experiences)
+      total_time = 0
+      breaks.each do |b|
+        volunteering_experiences_during_break(b, volunteering_experiences).each do |v|
+          total_time += volunteering_time_during_break(b, v)
+        end
+      end
+      convert_seconds_to_months(total_time)
+    end
+
+    def volunteering_time_during_break(work_break, volunteering_experience)
+      volunteering_end_date = volunteering_experience.end_date ||= Time.zone.now
+
+      start_date = work_break.start_date < volunteering_experience.start_date ? volunteering_experience.start_date : work_break.start_date
+      end_date = work_break.end_date > volunteering_end_date ? volunteering_end_date : work_break.end_date
+      end_date - start_date
+    end
+
+    def breaks_in_last_five_years(breaks, application_form)
+      breaks.count { |b| b.start_date > (submitted_at(application_form) - 5.years).to_date }
+    end
+
+    def unexplained_breaks_that_coincide_with_degrees(application_form, unexplained_breaks)
+      degrees = application_form.application_qualifications.degrees
+      unexplained_breaks.count { |unexplained_break| unexplained_break_coincides_with_a_degree(unexplained_break, degrees) }
+    end
+
+    def unexplained_break_coincides_with_a_degree(unexplained_break, degrees)
+      degrees.select { |degree|
+        coincides?(Date.new(degree.start_year.to_i, 1, 1), Date.new(degree.award_year.to_i, 12, 31), unexplained_break.start_date, unexplained_break.end_date)
+      }.any?
+    end
+
+    def breaks_that_coincide_with_volunteering_experiences(breaks, volunteering_experiences)
+      breaks.count { |b| volunteering_experiences_during_break(b, volunteering_experiences).any? }
+    end
+
+    def volunteering_experiences_during_break(work_break, volunteering_experiences)
+      volunteering_experiences.select do |volunteering_experience|
+        volunteering_end_date = volunteering_experience.end_date ||= Time.zone.now
+        coincides?(volunteering_experience.start_date, volunteering_end_date, work_break.start_date, work_break.end_date)
+      end
+    end
+
+    def coincides?(event_1_start_date, event_1_end_date, event_2_start_date, event_2_end_date)
+      event_1_start_date < event_2_end_date && event_2_start_date < event_1_end_date
+    end
+
+    def breaks_with_over_fifty_percent_volunteering(breaks, volunteering_experiences)
+      breaks.count { |b| volunteering_percentage_in_break(b, volunteering_experiences) > 0.5 }
```

Comment:
```suggestion
      breaks.count { |break| volunteering_percentage_in_break(break, volunteering_experiences) > 0.5 }
```

We should avoid using single character variables unless absolutely necessary, to promote better readability throughout the codebase.

Reply:
I slipped back to old habits because `break` is a keyword in Ruby and so it can't be used here. You're right, I'll change it to `work_break` here